atomicapp/providers/README.md

```diff
@@ -0,0 +1,101 @@
+# Providers
+
+Provider is a way for **atomicapp** to interact with a container engine.
+Currently, **atomicapp** has support for multiple providers:
+- docker
+- kubernetes
+- openshift
+- docker-compose
```

Comment:
not a big deal.. he can remove from here and then we'll merge and then he can add it to his compose PR